spec/models/defence_request_spec.rb

```diff
@@ -1,6 +1,24 @@
 require "rails_helper"
 
 RSpec.describe DefenceRequest, type: :model do
+  describe "state_updated_at" do
+    subject { FactoryGirl.create :defence_request }
+
+    it "is set equal created_at when creating" do
+      expect(subject.state_updated_at).to be_within(1.second).of(subject.created_at)
```

Comment:
don't like it, can we stub time or something instead of assuming that these tests will run quickly?